hibernate-redis/src/main/java/org/hibernate/cache/redis/jedis/JedisClient.java

```diff
@@ -487,6 +487,15 @@ public long nextTimestamp(final Object key) {
         }
     }
 
+    /**
+     * Cleanup any resources thathe JedisClient might have references to.
+     */
+    public void destroy() {
+        if (jedisPool != null) {
```

Comment:
not sure if this could ever be null but I guess it doesn't hurt to have the check.